src/util/sss_ssh.h

```diff
@@ -51,6 +51,7 @@ sss_ssh_format_pubkey(TALLOC_CTX *mem_ctx,
                       char **result);
 
 errno_t
-sss_ssh_print_pubkey(struct sss_ssh_pubkey *pubkey);
+sss_ssh_print_pubkey(struct sss_ssh_pubkey *pubkey,
+                     const char *_keyhost);
```

Comment:
Why '_' at the beginning of  '_keyhost'? In SSSD code base it is typically used for output parameters.

Reply:
It's a typo. The function definition doesn't have it.

Fixed.